### ISSUE TITLE
CommandBar Accessibility: JAWS reader says 'to move through items press up or down arrow', while those commands don't work'

### ISSUE THREAD

**Issue by nitikasaran68** [OfficeDev/office-ui-fabric-react#8024]
I'm using the CommandBar component in my app, and when I use the JAWS screen reader on the bar it says "to move through items press up or down arrow", which is the default behavior for role="menubar" elements in JAWS. However since I think fabric overrides all arrow key navigation on the CommandBar, this doesn't work. So the user might be confused as to how to navigate.
### Bug Report

- __Package version(s)__: 6.106.3
- __Browser and OS versions__: (fill this out if relevant)

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? (Yes, No)

Requested priority: (Blocking, High, Normal, Low) 

Products/sites affected: (if applicable)

**Comment by aneeshack4** [comment]
Thanks for reporting this! This behavior isn't fully correct in Narrator either. As seen in the NarratorBuddy output below, it says "use left & right arrow keys to navigate" but up & down are the ones that work, left & right do nothing & result in Narrator saying "No next text".
![image](https://user-images.githubusercontent.com/4161791/53038799-72c47e80-3432-11e9-92b8-bf4ac9ffc32e.png)

**Comment by aneeshack4** [comment]
This behavior works correctly in Edge + Narrator: 
![image](https://user-images.githubusercontent.com/4161791/53043831-40208300-343e-11e9-9558-3ab0e1bd4bd6.png)

So this is probably a Chrome and/or JAWS issue - we support Edge + Narrator primarily.
